sample-app/build.gradle

```diff
@@ -1,15 +1,15 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 26
-    buildToolsVersion "26.0.1"
+    compileSdkVersion 28
+    buildToolsVersion "28.0.3"
 
     defaultConfig {
         applicationId "com.tokbox.android.IBSample"
         minSdkVersion 16
-        targetSdkVersion 26
-        versionCode 1
-        versionName "2.0.0"
+        targetSdkVersion 28
+        versionCode 2
+        versionName "3.0.0"
```

Comment:
@ggoldens should it be 3.0.0? what is the semantic versioning we were using for IBS?

Reply:
You're right, I changed it, could you please 2x check?